src/test/kotlin/me/serce/solidity/lang/core/resolve/SolAliasResolveTest.kt

```diff
@@ -0,0 +1,156 @@
+package me.serce.solidity.lang.core.resolve
+
+class SolAliasResolveTest : SolResolveTestBase() {
+    fun testResolveImportedFunctionFromBracketWithAlias() = testResolveBetweenFiles(
+        InlineFile(
+            code = """
+          contract a {
+            function doit() {
+                     //x
+            }
+          }
+      """,
+            name = "a.sol"
+        ),
+        InlineFile(
+            """
+          import {a as A} from "./a.sol";
+              
+          contract b {
+            function test() {
+                A.doit();
```

Comment:
this example compiles by solc with error: `TypeError: Cannot call function via contract type name.`
Let's change it (and other tests) to make a real use case.

Reply:
Yes thank you for this feedback, I found more use cases. I'm gonna fix them